Source/DafnyCore/AST/TopLevelDeclarations.cs

```diff
@@ -595,7 +595,9 @@ public class ModuleQualifiedId {
       this.Sig = null;
     } else {
       this.Decl = m;
-      this.Def = ((LiteralModuleDecl)m).ModuleDef;
+      this.Def = m is AliasModuleDecl ?
+        m.Signature.ModuleDef :
+        ((LiteralModuleDecl)m).ModuleDef;
```

Comment:
Could we always use `m.Signature.ModuleDef`, or is there a way to make this consistent without having a special case?  Otherwise I'm not sure how to review this :/

Reply:
> Could we always use m.Signature.ModuleDef, or is there a way to make this consistent without having a special case? Otherwise I'm not sure how to review this :/

Right ! That's a good observation. Since all modules have a signature and each signature has a module definition, what you suggest seems strictly better for maintainability.
I implemented the fix. Reviewing it should be easier now.